distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc

```diff
@@ -113,6 +113,13 @@ to:
 RUN_AS_USER=<${branding-lowercase}-user>
 ----
 +
+. Set the `DDF_HOME` property.
```

Comment:
before it had to be DDF but I actually may be able to remove this whole section since I removed the line that used DDF_HOME environment variable.